Scripts/RemediationScripts/Remediate-DisableNonSSLPortOnRedisCache.ps1

```diff
@@ -306,10 +382,14 @@ function Disable-NonSSLPortOnRedisCache
     if ($totalRedisCacheWithNonSSLPortEnabled  -eq 0)
     {
         Write-Host "No Redis Cache(s) found with Non-SSL port enabled.. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
-        break
+        Write-Host $([Constants]::DoubleDashLine)
```

Comment:
Since here we are returning the control back/ending execution.

Though we have 0 redis cache having non ssl port enabled but if we have any redis cache resource we should show that in log file right? as they all skipped.

So add that code block here => 

if($AutoRemediation -and $totalCloudService -gt 0) 
        {
            $logFile = "LogFiles\"+ $($TimeStamp) + "\log_" + $($SubscriptionId) +".json"
            $log =  Get-content -Raw -path $logFile | ConvertFrom-Json
            foreach($logControl in $log.ControlList){
                if($logControl.ControlId -eq $controlIds){
                    $logControl.RemediatedResources=$logRemediatedResources
                    $logControl.SkippedResources=$logSkippedResources
                }
            }
            $log | ConvertTo-json -depth 100  | Out-File $logFile
        }

similar to this